control/control.go

```diff
@@ -302,18 +302,14 @@ func translateLegacySolveRequest(req *controlapi.SolveRequest) error {
 		req.Cache.Imports = append(req.Cache.Imports, im)
 	}
 	req.Cache.ImportRefsDeprecated = nil
-	return nil
 }
 
 func (c *Controller) Solve(ctx context.Context, req *controlapi.SolveRequest) (*controlapi.SolveResponse, error) {
 	atomic.AddInt64(&c.buildCount, 1)
 	defer atomic.AddInt64(&c.buildCount, -1)
 
 	// This method registers job ID in solver.Solve. Make sure there are no blocking calls before that might delay this.
```

Comment:
unrelated to this PR changes, but I wonder what line this comment is for. We should remove it or move it to the right place.

Reply:
thanks @tonistiigi , i removed this comment, cannot find which function this comment is for.

Reply:
Looks like it was from https://github.com/moby/buildkit/pull/2369/files#diff-0397d42d692ad676389ccf4b5a1126a3976ef69d4342b8423a6bbb51d21a9181R241.

My guess is that the comment is meant to apply to the whole of `Controller.Solve`?